codalab/lib/bundle_cli.py

```diff
@@ -1081,9 +1110,13 @@ def do_run_command(self, args):
         ),
     )
     def do_edit_command(self, args):
-        client, worksheet_uuid = self.parse_client_worksheet_uuid(args.worksheet_spec)
-        bundle_uuid = worksheet_util.get_bundle_uuid(client, worksheet_uuid, args.bundle_spec)
-        info = client.get_bundle_info(bundle_uuid)
+        client, worksheet_uuid = self.parse_client_worksheet_uuid(args.worksheet_spec, use_rest=True)
+
+        info = client.fetch('bundles', params={
+            'specs': args.bundle_spec,
+            'worksheet': worksheet_uuid,
+        })[0]
```

Comment:
We should have a client.fetch_one method that ensures that there's exactly one or throws a nice error otherwise.